src/sass/variables.scss

```diff
@@ -0,0 +1,12 @@
+@charset "UTF-8";
```

Comment:
I changed it to "shared-variables," I think that's a reasonable name and I'll add some comments.